test/nbody_water_test.jl

```diff
@@ -1,78 +1,160 @@
-println("====  Water SPC/Fw test   ====")
-let
+@testset "Water SPC/Fw test" begin
     qe = 1.6e-19
     Na = 6.022e23
     T = 298.16 # °K
     kb = 1.38e-23 # J/K
-    ϵOO = 0.1554253*4184/Na
+    ϵOO = 0.1554253 * 4184 / Na
     σOO = 3.165492e-10 # m
     ρ = 900 # kg/m^3
     mO = 15.999 * 1.6747 * 1e-27 # kg
```

Comment:
Is this test working at all? I can't see the units of 1e23 with 1e-27 not cancelling out to zero?

Reply:
Actually, it works: I tested it manually with this units, though it can be considered as the continuation of our previous discussion about advantages of OpenMM units. Such tests with SI units are on my list for cleaning up the project. I'll change it in NBodySimulator.

`ϵOO ~ 1e-21` and together with distance measured in angstroms or `1e-10 m` gives us a force of the order `1e-11`.  So the acceleration is about `1e16 m/s`